user-org-util/pom.xml

```diff
@@ -23,6 +23,16 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.9</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.9.8</version>
```

Comment:
Use 2.9.9 and preferably take it from parent.pom, if it exists. Or create a property locally.

Reply:
fixed.